app/helpers/application_helper.rb

```diff
@@ -76,6 +76,28 @@ def format_datetime(date, format = :long)
   def time_period_class(item)
     return 'not-yet-valid' if item.not_yet_valid?
     return 'currently-valid' if item.currently_valid?
-    return 'expired' if item.expired?
+    'expired' if item.expired?
+  end
+
+  # Generates a page header. The title shown will be the +.header+ key in the page
+  # that calls this helper. Buttons will only be shown if the current user has the
+  # +create+ privilege.
+  # @param [Class] resource The resource header buttons will manage. If left unspecified,
+  #   the other options will be ignored.
+  # @param [Hash] params Helper methods to call, the results of which will be placed in a div
+  # positioned on the far right. Keys will be interpreted as method names, values as arguments.
+  def page_header(resource=nil, params={})
```

Comment:
Method has too many lines. [12/10]<br>Surrounding space missing in default value assignment.